Formula/notmuch-mutt.rb

```diff
@@ -4,6 +4,7 @@ class NotmuchMutt < Formula
   url "https://notmuchmail.org/releases/notmuch-0.32.2.tar.xz"
   sha256 "8e0a7eb8ff2e6011ef48b2bf11d79b9c4bb74511cfe2987758b64898c2a2ded7"
   license "GPL-3.0-or-later"
+  revision 1
```

Comment:
Currently (while merging `linuxbrew-core` and `homebrew-core`), we don't update the revision for Linux-only changes. Once PR is merged, we can bottle a copy in `homebrew-core` and a maintainer in `linuxbrew-core` can do the same there.